test/SlowTests/Issues/RavenDB-17745.cs

```diff
@@ -86,6 +84,10 @@ public async Task StartStoreInTheMiddleOfAnHeartbeat()
                 using (var session = store.OpenSession())
                 {
                     var user = session.Load<User>("users/1");
+                    if (user == null)
+                    {
+                        user = session.Load<User>("users/1");
```

Comment:
Shouldn't the document be saved after `bulk.Dispose();`?

Reply:
yes